socialdistribution/bettersocial/views.py

```diff
@@ -176,6 +192,38 @@ def post(self, request, uuid, action, *args, **kwargs):
         elif action == 'unfollow':
             Following.objects.filter(following_uuid = uuid, author = author).delete()
             Follower.objects.filter(follower_uuid = author.uuid, author_id = uuid).delete()
+
+        # If target user is not local, need to send api request
+        if not Author.objects.filter(uuid = uuid).exists():
+            node = Node.objects.filter(host__contains = self.request.GET['host']).get()
+
+            author_serialized = AuthorSerializer(author, context = { 'request': self.request }).data
+            follower_json = {
+                'type': 'author',
+                'id': author_serialized['id'],
+                'url': author_serialized['url'],
+                'host': author_serialized['host'],
+                'displayName': author.display_name,
+                'github': author.github_url,
+                'profileImage': author_serialized['profileImage'] or ''
+            }
+
+            # TODO: need foreign author uuid with remote host url
+            url = (yarl.URL(node.host) / node.prefix / 'author' / author.uuid / 'followers' / uuid).human_repr()
+            response = requests.post(
+                url,
+                headers = { 'Accept': 'application/json' },
+                auth = HTTPBasicAuth(node.node_username, node.node_password),
+                json = follower_json
+            )
```

Comment:
I just realized that [Abram said that he does not want followers created via POST to `.../followers`.](https://eclass.srv.ualberta.ca/mod/forum/discuss.php?d=1827396) Instead, we should send that event to the inbox for the remote server to handle.

Unless I am mistaken, `POST /author/<uuid>/followers` is internal use only.

for the record the correct JSON for the `.../inbox` POST is
```json
{
    "type": "Follow",
    "summary": "Local User wants to follow Remote User",
    "actor": {
        "type": "author",
        # Our User
    },
    "object": {
        "type": "author",
        # Their User
    },

}
```